5.4.Poter_Book.py

```diff
@@ -0,0 +1,22 @@
+import os
+import re
+
+def rename_chapters(folder_path):
```

Comment:
Separate code to small functions- 
https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be

This code is not clean